tool/plugin/lib/plugin.dart

```diff
@@ -589,21 +591,21 @@ https://plugins.jetbrains.com/plugin/uploadPlugin
   }
 }
 
-/// Generate the plugin.xml from the plugin.xml.template file.
-/// If the --release argument is given, create a git branch and
-/// commit the new file to it, assuming the release checks pass.
+/// Generate the plugin.xml from the plugin.xml.template file. If the --release
+/// argument is given, create a git branch and commit the new file to it,
+/// assuming the release checks pass.
 ///
-/// Note: The product-matrix.json file includes a build spec for the
-/// EAP version at the end. When the EAP version is released that needs
-/// to be updated.
+/// Note: The product-matrix.json file includes a build spec for the EAP version
+/// at the end. When the EAP version is released that needs to be updated.
 class GenerateCommand extends ProductCommand {
   final BuildCommandRunner runner;
 
   GenerateCommand(this.runner) : super('generate');
 
-  String get description =>
-      'Generate a valid plugin.xml and .travis.yml for the Flutter plugin.\n'
-      'The plugin.xml.template and product-matrix.json are used as input.';
+  String get description => 'Generate plugin.xml, .travis.yml, and '
```

Comment:
Nit: we don't use travis anymore. We generate the github actions file now.

Reply:
updates